pkg/ddevapp/provider.go

```diff
@@ -15,11 +15,13 @@ import (
 
 // ProviderCommand defines the shell command to be run for one of the commands (db pull, etc.)
 type ProviderCommand struct {
-	Command string `yaml:"command"`
-	Service string `yaml:"service,omitempty"`
+	Command    string `yaml:"command"`
+	Service    string `yaml:"service,omitempty"`
+	SkipImport bool   `yaml:"skip_import"`
```

Comment:
Wouldn't it make sense to add a files_import_command and a db_import_command, which could be empty?

I guess that leads to potential backward-compatibility problems, but it's worth thinking about.

Perhaps add an import-api version to solve that?

Overall, I think the actual db import logic and files import logic should be moved into the yaml file. What you you think of that?

Reply:
I understand you mean the actual last "import into ddev" step, do you?

I wonder if it's worth it to provide an API catering for all scenarios like multiple databases/file locations and multiple artifact formats.

btw: I am not sure I defined that correctly here. What I wanted to achieve: `SkipImport` shall be an optional boolean, defaulting to `false`. I just found out through testing that it worked like that.

Reply:
Right, I'm saying actually define the import-into-ddev (db and files) steps as separate items. 

Skipping the import isn't exactly what we're trying to accomplish, right? We're trying to *delegate* the import. So I'm saying why not just always delegate it to a script?

Reply:
I like where you're going... and the "leave out for default" solves so many problems.

I agree that pull implies everything... but `pull` has a long history in ddev and it wouldn't probably be worth changing it.